Rakefile

```diff
@@ -7,12 +7,8 @@ RSpec::Core::RakeTask.new(:spec) do |task|
   task.rspec_opts = '--color'
 end
 
-task :build do
-  sh 'bundle exec jekyll build --config _config.yml,_config_en.yml --trace'
-end
-
 task :buildboth do
-  sh 'bundle exec jekyll build --trace'
+  sh 'bundle exec jekyll build --config _config.yml,_config_en.yml --trace'
```

Comment:
Exactly. Why not keep the one from the readme people are used to?  You're breaking the interface